catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/operation/MysqlTableOperations.java

```diff
@@ -48,46 +43,95 @@ public class MysqlTableOperations extends JdbcTableOperations {
 
   @Override
   public JdbcTable load(String databaseName, String tableName) throws NoSuchTableException {
-    CreateTable createTable = loadCreateTable(databaseName, tableName);
     List<JdbcColumn> jdbcColumns = new ArrayList<>();
-    // Assemble index information.
-    Map<String, Set<String>> indexGroupByName =
-        getIndexNameGroupByColumnName(createTable.getIndexes());
-    for (ColumnDefinition columnDefinition : createTable.getColumnDefinitions()) {
-      // Assemble column information.
-      String columnName = getColumnName(columnDefinition);
-      String[] columnSpecs =
-          columnDefinition.getColumnSpecs() == null
-              ? new String[0]
-              : columnDefinition.getColumnSpecs().toArray(new String[0]);
-      String columnProperties = String.join(SPACE, columnSpecs);
-      boolean nullable = !columnProperties.contains(NOT_NULL);
-      String defaultValue = findPropertiesValue(columnSpecs, DEFAULT);
-      String comment = findPropertiesValue(columnSpecs, COMMENT);
-      List<String> properties = getColumnProperties(columnProperties);
-      Optional.ofNullable(indexGroupByName.get(columnName)).ifPresent(properties::addAll);
-      jdbcColumns.add(
-          new JdbcColumn.Builder()
-              .withName(columnName)
-              .withType(typeConverter.toGravitinoType(columnDefinition.getColDataType()))
-              .withNullable(nullable)
-              .withComment(comment)
-              // TODO: uncomment this once we support column default values.
-              // .withDefaultValue("NULL".equals(defaultValue) ? null : defaultValue)
-              .withProperties(properties)
-              .build());
+    try (Connection connection = getConnection(databaseName)) {
+      // 1.Get table information
+      DatabaseMetaData metaData = connection.getMetaData();
+      ResultSet table = metaData.getTables(databaseName, null, tableName, null);
```

Comment:
why `schemaPattern` is null when `getTables`? I wonder maybe getting multi-tables if not specify schemaPattern;

Reply:
ok